src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoresSpec.scala

```diff
@@ -227,6 +228,24 @@ class BagStoresSpec extends TestSupportFixture
     }
   }
 
+  it should "fail if the bag was marked as inactive" in {
+    implicit val baseDir: BaseDir = bagStore1.baseDir
+    bagStore1.add(testBagMinimal) should matchPattern {
```

Comment:
I would suggest using `inside` here, rather than `matchPattern`:

```scala
inside(bagStore1.add(testBagMinimal)) {
  case Success(bagId: BagId)) =>
    bagStore1.deactive(bagId) shouldBe a[Success[_]]
    bagStore1.enumFiles(bagId) should matchPattern { case Failure(InactiveException(bagId)) => }
}
```

same in the test below

Reply:
@jo-pol  now i am confused

Reply:
I think I understand. Inside is to be avoided to check the result of a single action as it is not clear what when wrong in case of failure. I suppose `bagStore1.add(testBagMinimal)` is a preparation for the actual test(s). 

Still I don't like the inside. Following quite recently adopted conventions:
`bagStore1.add(testBagMinimal).getOrRecover(e => fail("could not prepare the bag", e))`